Maple2Storage/Scripts/11000024.lua

```diff
@@ -0,0 +1,6 @@
+function getFirstScriptId()
```

Comment:
Can we have all these NPC scripts in a sub folder?

Reply:
Sure!